snuba/cli/subscriptions.py

```diff
@@ -150,7 +160,10 @@ def subscriptions(
         SubscriptionResultCodec(),
     )
 
-    with closing(consumer), closing(producer):
+    executor = ThreadPoolExecutor(max_workers=max_query_workers)
+    logger.debug("Starting %r with %s workers...", executor, executor._max_workers)
```

Comment:
I don't like using the underscore-prefixed variable name here, but there's no other way to get the default value of this limit otherwise when using the default value.